set all in RegisterForm

vanya-n23 opened the pull request.

vanya-n23 commented:
Make all needed on RegisterForm with Render component on RegisterPAge

vanya-n23 committed:
set all in RegisterForm

DarkLordSith reviewed (changes requested):
- Налаштувати валідацію поля email, щоб він не приймав email якщо попереду нього стоїть пробіл (або автоматично видаляв зайві пробіли при знятті фокусу з поля email)
- Відкоригувати позиціонування повідомлення від hottoast відносно хедеру (зараз він закриває повідомлення) наприклад змінити inset, у стилях повідомлення, на 90px
- Звірити я із загальними стилями і прибрати/відредагувати зайві стилі до компонента
- Не зрозуміло стосовно motion, воно ні як не використовується в коді RegistrationForm

vanya-n23 commented:
Додав зміни які треба було і виправив стилі

vanya-n23 commented:
1

vanya-n23 committed:
add all notfication for RegisterForm

vanya-n23 committed:
set without header

vanya-n23 committed:
updadet bakcend call

vanya-n23 committed:
api update

vanya-n23 committed:
updated backend for registerfORM

змінив також в арі основну силку на рендер нашого бекенду, то ти вже тільки прийми і змержи цей коміт і для інших воно буде добре напевно, просто вже не треба буде тобі то міняти (напевно) тому все поки добре відносно

vanya-n23 committed:
1
